### PR TITLE
[BE]: Add tensor to constant comparison test

### PR DESCRIPTION
**Pull request by Skylion007**
Add additional test of tensor to constant comparisons. We already compare constant to tensor.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang

### CODE REVIEW

**Review by albanD — APPROVED** [review]
More tests :tada:

File: test/dynamo/test_functions.py [hunk 1]
```diff
@@ -2010,6 +2013,18 @@ def fn(x):
...
                 self.assertEqual(opt_fn(x), fn(x))
 
+    def test_compare_tensor_and_constant(self):
+        for op in comparison_ops:
+            with self.subTest(op=op):
+
+                def fn(x):
+                    return op(x, -10)
+
+                opt_fn = torch.compile(fullgraph=True)(fn)
```
Comment (zou3519) [inline test/dynamo/test_functions.py:2023]: Use torch._dynamo.optimize or torch.compile(backend="eager"). This is test/dynamo -- we don't generally run things through all of PT2 stack in tests in this directory.

**Review by zou3519 — CHANGES_REQUESTED** [review]
See comment

**Review by zou3519 — APPROVED** [review]

**Pull request closed without merging**